client/src/css/Products.css

```diff
@@ -1,57 +1,98 @@
+
 #product{
-    width: 100%;
-    display: flex;
-    justify-content: space-around;
-    flex-wrap: wrap;
+  width: 100%;
+  display: flex;
+  justify-content: space-around;
+  flex-wrap: wrap;
 }
 .card{
-    min-width: 300px;
-    min-height: 400px;
-    border: 1px solid #eee;
-    overflow: hidden;
-    padding: 10px;
-    box-shadow: 2px 8px 20px #ddd;
-    margin: 10px;
-    transition: 0.5s linear;
+  min-width: 300px;
+  min-height: 400px;
+  border: 1px solid #eee;
+  overflow: hidden;
+  padding: 10px;
+  box-shadow: 2px 8px 20px #ddd;
+  margin: 10px;
+  transition: 0.5s linear;
 }
 .card:hover{
-    box-shadow: none;
+  box-shadow: none;
 }
 .card img{
-    max-width: 300px;
-    width: 100%;
-    height: 100%;
-    max-height: 300px;
-    display: block;
-    object-fit: cover;
+  max-width: 300px;
+  width: 100%;
+  height: 100%;
+  max-height: 300px;
+  display: block;
+  object-fit: cover;
 }
 .card h3{
-    text-transform: uppercase;
-    margin: 10px 0;
+  text-transform: uppercase;
+  margin: 10px 0;
 }
 .card h3 a{
-    text-decoration: none;
-    color: #333;
+  text-decoration: none;
+  color: #333;
 }
 .card h3 a:hover{
-    color: crimson;
+  color: crimson;
 }
 .card span{
-    color: crimson;
+  color: crimson;
 }
 .card p{
-    margin: 10px 0;
+  margin: 10px 0;
 }
 .card button{
-    border: none;
-    outline: none;
-    background: #333;
-    color: white;
-    width: 100%;
-    height: 40px;
-    display: block;
-    cursor: pointer;
-    text-transform: uppercase;
-    letter-spacing: 2px;
-    margin: 15px 0;
+  border: none;
+  outline: none;
+  background: #333;
+  color: white;
+  width: 100%;
+  height: 40px;
+  display: block;
+  cursor: pointer;
+  text-transform: uppercase;
+  letter-spacing: 2px;
+  margin: 15px 0;
+}
+
+
+
+
+.row{
+  margin-left: 200px;
+  float: left;
+  /* width: 200; */
```

Comment:
if you leave commented code then you should leave a comment why this code should stay commented. Otherwise we can store a lot of commented code during development and after a month, two or three we will not remember why we left it and did not remove.